lib/wordpress.js

```diff
@@ -0,0 +1,53 @@
+var _ = require('underscore');
+var qs = require('querystring');
+var util = require('./util');
+
+module.exports = {
+  params: !util.isNode?qs.parse(window.location.search.split('?').pop()):{},
+  getPostPreviewView: function (view) {
+    // Return false if we're not in preview mode
+    if (!this.params.is_preview) {
```

Comment:
Unfortunately `this` will be wrong if you use the library like this:
```javascript
var getPostPreviewView = require('solidus-client/lib/wordpress').getPostPreviewView;
getPostPreviewView(view); // BLAMMO
```

You could call it like this instead: `if (!module.exports.params.is_preview) {`

Or, move the function to the util library (so it can be mocked during the tests):
```javascript
  getPostPreviewView: function (view) {
    // Return false if we're not in preview mode
    var params = util.params();
    if (!params.is_preview) {
```